docs/guides/serverless-hamsim-template.ipynb

```diff
@@ -0,0 +1,1033 @@
+{
```

Comment:
<p>I'm assuming this second item <code>QiskitFunction(aqc_tutorial_L50)</code> should not be here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>I just manually deleted it since I can't run the notebook.  LMK if we need to put it back. </p>